### DISCUSSION TITLE
cr-touch for an Ender-3 that has been modified with a V2 display and Sprite Extruder

### DISCUSSION THREAD

**Discussion by doctorevil30564** [mriscoc/Ender3V2S1#869] [General]
Hey folks, I used the source code to create custom firmware using the march 2023 release of the firmware. On the machine this is running on the setup is as follows

Original Ender 3 printer with a v4.2.2 mainboard
Sprite Extruder Pro kit installed with a cr-touch ABL probe installed
Ender 3 V2 display flashed with the necessary firmware to properly display using the mriscoc ender3v2S1 firmware

I used the 256k v4,2,2 board definition  to ensure that whatever version of the 4.2.2 board I have would work.

After determining the optimal positioning to center the probe on the bed for homing and doing a G29 command (I didn't use the UBL bed level I went with the bilinear configuration), when I first power on the printer the probe works as it should, but if I do not turn off the printer after finishing a print, and then try to do another print, the probe fails to deploy, so even just doing the regular z axis homing results in the print nozzle hitting the print bed and knocking the x gantry out of alignment in relation to the frame of the printer.

Has anyone else had this issue and were you able to fix the problem?

I do have a spare cable to connect the probe to the sprite extruder PCB connector, but if it's something wrong with how I have my config setup for compiling the firmware, I would like to fix it rather than try the cable and then still have problems with it.

Other than this issue, the firmware works well, and I get incredibly good prints off the printer so long as I always power cycle the printer before trying another print.

Thanks in advance for any info you can share to help me with fixing this issue.

**Comment by mriscoc** [comment]
Be sure that the HS mode was not changed in the source code.
